pom.xml

```diff
@@ -695,6 +702,26 @@
                 <scope>test</scope>
             </dependency>
 
+            <dependency>
+                <groupId>com.github.oowekyala.treeutils</groupId>
+                <artifactId>tree-matchers</artifactId>
+                <version>2.1.0</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>com.github.oowekyala.treeutils</groupId>
+                <artifactId>tree-printers</artifactId>
+                <version>2.1.0</version>
+                <scope>test</scope>
+            </dependency>
+
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>18.0</version>
```

Comment:
What's the reason of using such an old guava? (or should I ask 'what's the reason of using guava at all' :))

Reply:
It's only used by pmd-apex-jorje - on master, it's already removed from the main parent pom.